assessments/projects/tikv/self-assessment.md

```diff
@@ -0,0 +1,335 @@
+   - ***TiKV Backup and Restore Configurations***: TiKV Backup & Restore ([TiKV-BR](https://tikv.org/docs/dev/concepts/explore-tikv-features/backup-restore/)) is a command-line tool for distributed backup and restoration of TiKV cluster data. Certain aspects of a TiKV backup can be configured including use of a checksum and use of TLS.
+
+
+
+## Project compliance
+
+Although TiKV does not provide explicit documentation regarding its compliance with specific regulations, it adheres to general best practices in security as an open-source distributed database. Although initially developed for the Chinese market, TiKV's documentation does not explicitly address its compliance with Western regulations like GDPR and HIPAA. The documentation of project compliance can potentially expedite the process of adoption.
+
+## Secure development practices
+
+For testing, TiKV uses regression tests, performance tests, ChaosMesh, and models their algorithms utilizing TLA+. TiKV uses Cargo as a universal project tool, and split tests into test modules in respective code files. Among the fuzzing libraries used are LLVM's libfuzzer, AFL and Google's Honggfuzz; however, tests do not run in an automated fashion.
+
+The TiKV project utilizes persistent open groups which each focus on a part of the TiKV project; teams and members provided in the teams folder for each individual project. Each team consists of reviewers, committers, and maintainers:
+* **Reviewers**: Responsible for contributions and code review 
+* **Committees**: Granted write access and usually responsible for certain areas of the project
+  - Each commit message must contain a “Signed-off-by” line for Developer Certificate of Origin
+* **Maintainers**: Responsible for moderating the discussions, managing project release, and proposing new committers or maintainers
+
+In addition, there is a separate infrastructure team, not responsible for the team decision-making process, but that helps on applying the decision. The current list of infrastructure members is provided with their emails and githubs on the TiKV Governance page.
+* **Infrastructure team**: Responsible for setting administration, security, and supporting the existing teams
+
+**Voting**
+* Decisions regarding the project are made by votes on the primary project community repository.
+* Votes are indicated via pull requests under the votes folder.
+* Positive votes require no explanation, while vetoes (negative votes) must have an explanation. 
+* Required vote counts can vary based on the decision but typically require at least 2 positive votes.
+
+* Internal Communication: Slack 
+
+ * Inbound Communication: Twitter, Blog, Reddit, Stack Overflow
+
+* Outbound Communication: None
```

Comment:
This section needs further elaboration. Can you please provide link/reference to these communication modes? A good reference is https://github.com/cncf/tag-security/blob/main/assessments/projects/pixie/self-assessment.md#communication-channels

Reply:
We closely followed the format of the example you provided. Thank you for the guidance!